bears/haskell/HaskellLintBear.py

```diff
@@ -38,11 +38,15 @@ def process_output(self, output, filename, file):
         output = json.loads(output)
 
         for issue in output:
-            assert issue['startLine'] == issue['endLine']
             diff = Diff(file)
+            from_lines = issue['from'].splitlines()
+            to_lines = issue['to'].splitlines()
+            assert len(from_lines) == len(to_lines)
+            for other_lines in range(1, len(from_lines)):
+                assert from_lines[other_lines] == to_lines[other_lines]
```

Comment:
If _ONLY_ all the other lines in the error message are equal (which is the case most of the times, I haven't come across a case where it is NOT equal till now, but it is better to keep the `assert` just in case), then we can output the change which is just the first line ^^
Then, we can apply the patch and change only the first line :smile: 
Otherwise, if the other lines lines are different, then we _CANNOT_ apply the patch